Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -157,7 +187,7 @@ namespace Hazel {
 	};
 }
 
-#define HZ_PROFILE 0
+#define HZ_PROFILE 1
```

Comment:
Do not enable profiling in a PR, only for testing
```suggestion
#define HZ_PROFILE 0
```

Reply:
Done.